src/libs/utils.js

```diff
@@ -48,3 +51,16 @@ export const download = (fileName, text) => {
   a.download = fileName + '-restriction';
   a.click();
 };
+
+export const Notifications = {
+  showError: error => {
+    return new Noty({
+      type: 'error',
+      layout: 'bottomRight',
+      theme: 'duos',
+      text: `Something went wrong. Please try again.\nError code: ${error.status}`,
+      timeout: '3500',
+      progressBar: false,
+    }).show();
```

Comment:
What I'd like to see here is an ability to customize the message. Absent arguments, this is a great default. To be more useful outside of the current context, I would introduce some props here.  Something like this (caveat ... completely untested):
```
showError: (props) => {
    return new Noty({
      type: 'error',
      layout: 'bottomRight',
      theme: 'duos',
      text: props.message,
      timeout: props.timeout,
      progressBar: false,
    }
```
It would be nice if props were an object with defined, default properties so a caller could pass in merged props with default props maybe like this (caveat, also untested):
```
    catch (e) {
      const mergedProps = {...Notifications.defaultProps, ...{message: 'my error message'}};
      Notifications.showError(mergedProps);
    }
```

Reply:
Addendum ... callers probably shouldn't do the merge, that should happen inside `Notifications`. Instead, callers should just pass in what they specifically care about, i.e. `Notifications.showError({message: 'my error message'})`

Reply:
Hopefully this is a good solution... The order in which the objects are merged allows the caller to customize `text`, `timeout`, etc., but not properties like `type` and `theme`, which should be left to their defaults. I also kept the `defaultNotification` object pretty general in case we wanted to implement a `showSuccess` function or something similar in the future.

Reply:
I like this! 👍